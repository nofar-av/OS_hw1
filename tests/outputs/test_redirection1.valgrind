==28522== Memcheck, a memory error detector
==28522== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28522== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28522== Command: /home/omer/OS_wet1/smash
==28522== Parent PID: 28517
==28522== 
==28522== 
==28522== FILE DESCRIPTORS: 5 open at exit.
==28522== Open file descriptor 4: test_redirection1.tmp
==28522==    at 0x54EEC3E: open (open64.c:47)
==28522==    by 0x110B19: RedirectionCommand::execute() (Commands.cpp:426)
==28522==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28522==    by 0x118C97: main (smash.cpp:42)
==28522== 
==28522== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_redirection1.valgrind
==28522==    <inherited from parent>
==28522== 
==28522== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_redirection1.err
==28522==    <inherited from parent>
==28522== 
==28522== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_redirection1.out
==28522==    <inherited from parent>
==28522== 
==28522== Open file descriptor 0:
==28522==    <inherited from parent>
==28522== 
==28522== 
==28522== HEAP SUMMARY:
==28522==     in use at exit: 0 bytes in 0 blocks
==28522==   total heap usage: 66 allocs, 66 frees, 83,310 bytes allocated
==28522== 
==28522== All heap blocks were freed -- no leaks are possible
==28522== 
==28522== For counts of detected and suppressed errors, rerun with: -v
==28522== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
