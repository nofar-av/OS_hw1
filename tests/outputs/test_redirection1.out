smash> smash> smash pid is 5943
smash> smash: sending SIGKILL signal to 0 jobs:
